recipes/uncrustify/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "uncrustify" %}
+{% set version = "0.69.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/uncrustify/uncrustify/archive/{{ name }}-{{ version }}.tar.gz
+    sha256: 33bd97a07f7c4bd114874f73171aca220bf05c17108f8505a117b97374a347b6
+
+build:
+  number: 0
+  skip: true  # [osx]
```

Comment:
Can you try removing this skip? I bet the osx library will build fine given what the install script looks like.